src/GraphQL.Tests/Instrumentation/ApolloTracingTests.cs

```diff
@@ -68,23 +68,28 @@ public void extension_has_expected_format()
     public void serialization_should_have_correct_case(IGraphQLTextSerializer writer)
     {
         var trace = new ApolloTrace(new DateTime(2019, 12, 05, 15, 38, 00, DateTimeKind.Utc), 102.5);
-        var expected = @"{
+#if NET7_0_OR_GREATER
```

Comment:
To eliminate the framework specific expected string, can the `ms` fraction be changed to `102`?  In that case the formatting would match all .NET frameworks.

Reply:
Let's leave it as you have it now.  I would prefer to investigate further and determine why the change occurred, and then determine if we should change the serializer implementation to format the value in a defined manner regardless of the target framework.

Reply:
It seems to be the issue is in the DateTime formatting library:

```
DateTime dt = new DateTime(2019, 12, 05, 15, 38, 00, DateTimeKind.Utc);
dt = dt.AddMilliseconds(102.5);
Console.WriteLine($"{dt.ToString("O")}");
```

net 7.0 -> `2019-12-05T15:38:00.1025000Z`
net5.0, net6.0 -> `2019-12-05T15:38:00.1030000Z`

Reply:
Or perhaps `AddMilliseconds` does not retain the fractional portion of the milliseconds specified... ?

- https://github.com/dotnet/runtime/pull/73198

Looks like this line here:

```cs
return AddTicks((long)millis_double * TicksPerMillisecond);
```

should have been this:

```cs
return AddTicks((long)(millis_double * TicksPerMillisecond));
```

but they cleaned it all up for .NET 7.

Reply:
So we could ignore it, or we could change the call to `AddMilliseconds` to use `AddTicks` instead, which would fix the bug in .NET 6 and prior versions.

Reply:
:+1: